prow/plugins.yaml

```diff
@@ -44,9 +52,7 @@ lgtm:
   - istio
   review_acts_as_lgtm: true
 - repos:
-  - istio/api
```

Comment:
What is the intent here?

Reply:
Like most of our other repos, istio/api and istio/istio are being moved to the mergify model for handling merges and stuff, and we want people to just use the GitHub UI in the normal way.